docs/spec.md

```diff
@@ -618,6 +620,32 @@ It is invalid to split the configuration; if the `include` clause is used, no
 other keys can be specified in the `tusk.yml`, and the full task must be
 defined in the included file.
 
+### Interpreter
+
+By default, any command run will default to using `sh` as its interpreter. This
+can optionally be configured using shebang-like semantics using the
+`interpreter` clause at the top level:
+
+```yaml
+interpreter: node
+
+tasks:
+  hello:
+    run: console.log("Hello!")
+```
+
+Like a shebang, the interpreter must follow the form:
+
+```text
+<interpreter> [optional-arg]
+```
+
+Unlike a shebang, however, it is not a requirement that the interpreter is the
+absolute path to the executable, so long as it is available on the `PATH`.
+
+This value can also be using the `TUSK_INTERPRETER` environment variable. The
+environment variable has higher precidence than the configuration value.
```

Comment:
I think it should be opposite; a configuration inside the file _should_ override the environment. Thus if setting `TUSK_INTERPETER` to `zsh`, we won't try to use `zsh` to run a tusk file with `interpeter` set to `python`. 

Alternatively, perhaps there is no strong need for an environment variable at all. While it works OK now, it may make it even harder to introduce per-task interpreter settings later?

Doesn't matter too much to me though, if you think it's important to be able to override, thats _fine_. I would rather just edit the file, keeping it declarative.

Reply:
> Alternatively, perhaps there is no strong need for an environment variable at all. 

I think being conservative here is the right call. I'll remove environment variable configuration for now, and reconsider if there's a strong case for it in the future.